spec/PasswordPolicy.spec.js

```diff
@@ -0,0 +1,697 @@
+"use strict";
+
+const request = require('request');
```

Comment:
@bhaskaryasa 
Can you please use`request-promise` or `request-promise-native` instead of just `request`?